src/providers/google_live.py

```
@@ -2122,66 +2127,86 @@ async def cancel_response(self) -> None:
-        self._force_farewell_sent = False
-        self._last_audio_out_monotonic = None
-        self._user_end_intent = None
-        self._assistant_farewell_intent = None
-        self._model_text_buffer = ""
-        self._input_transcription_buffer = ""
-        self._output_transcription_buffer = ""
-
-        if self._session_start_time:
-            duration = time.time() - self._session_start_time
+        self._closing = True
+        previous_call_id = self._call_id
```

Ruff (0.15.0)

[error] Local variable `previous_call_id` is assigned to but never used

Remove assignment to unused variable `previous_call_id`

(F841)